### PR TITLE
link PR vignette from PR helpers manual page

### LINKED ISSUE #938: Doc suggestion: link PR vignette from PR helpers manual page
I was reading https://usethis.r-lib.org/reference/pr_init.html which unless I missed it doesn't mention https://usethis.r-lib.org/articles/articles/pr-functions.html which I nearly missed, so I'd suggest adding a link.

### PR DESCRIPTION
**Pull request by nandriychuk**
Fixes #938

### CODE REVIEW

**Comment by jennybc** [comment]
Can you edit to say "Fixes #938"? This is just a typo, but we should fix so this closes the correct issue.

File: R/pr.R [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #' (SSH vs HTTPS) is determined from the existing remote URL(s) of the repo.
 #'
 #' @section For contributors:
-#' To contribute to a package, first use `create_from_github(owner/repo)` to
+#' [To contribute to a package](https://usethis.r-lib.org/articles/articles/pr-functions.html), first use `create_from_github(owner/repo)` to
```
Comment (jennybc) [inline R/pr.R]: Can you add this link in a very obvious way? As more like, "to read more about blah blah blah, see the yada yada vignette".

Reasoning is along these lines:

https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

Please also follow the line length convention in surrounding lines.

**Comment by nandriychuk** [comment]
Added an additional sentence to make a link more obvious. 
Made edits to follow the length convention in surrounding lines.

**Comment by jennybc** [comment]
Thanks!

**Pull request merged**